tests/php/StaticPublisherTest/Model/DataObjectNoTrigger.php

```diff
@@ -7,9 +7,10 @@
 
 class DataObjectNoTrigger extends DataObject implements TestOnly
 {
+    private static $table_name = 'StaticPublishQueue_DataObjectNoTrigger';
```

Comment:
I've created https://github.com/silverstripe/silverstripe-standards/issues/15 to add a phpstan rule to resolve these going forward